### ISSUE TITLE
Cannot remove entity if ID disabled

### ISSUE THREAD

**Issue by ElinaSavina** [KorzhCom/EasyData#9]
I don't want to fill the ID property, because it will be generated. So I added `[MetaEntityAttr(Enabled = false)]` to not display it on the creation form. But now I can't remove this entity, because lib cannot get the Id. Please review my screenshots:
![image](https://user-images.githubusercontent.com/25950614/104725734-89210500-5743-11eb-86a6-51e17156f2a5.png)
![image](https://user-images.githubusercontent.com/25950614/104725627-655dbf00-5743-11eb-9b2b-6e8cb479850f.png)

**Comment by ElinaSavina** [comment]
Also, if I disable ID, a related entity is not shown in the creation form:
![image](https://user-images.githubusercontent.com/25950614/104733954-87aa0980-5750-11eb-8be6-536997b005bf.png)

**Comment by korzh** [comment]
Hello,

I think we will add 3 new properties boolean to the MetaEntityAttr attribute: ShowOnView, ShowOnEdit, ShowOnCreate. So it will be possible to show/hide any field either from view (grid), from the Edit form, or from the Create form. 
With this approach "Visible" property available now will become obsolete and will be replaced by ShowOnView.

Will this approach suit your requirements?

**Comment by antifree** [comment]
Hello,

We have published `1.2.0-rc1` version. Update packages to this version.
JS bundle should be updated to:
```html
<script src="https://cdn.korzh.com/ed/1.2.0/easydata.min.js" type="text/javascript"></script>
```

Now, you can hide any attribute in different scenarios using such options:
```c#
[MetaEntityAttr(ShowOnCreate = false, ShowOnEdit = false, ShowOnView = false)]
public string CategoryName { get; set; }
```

**Comment by ElinaSavina** [comment]
Hi, it works now. Thanks!
